### PR TITLE
feature: add shortcut m for toggle the move num of nodes

### PR DESCRIPTION
**Pull request by xiaoyifang**
press m to toggle the move num .

image

https://github.com/sanderland/katrain/assets/105986/7f5964ad-a468-48ff-846f-36c2148745ae

### CODE REVIEW

File: katrain/gui/badukpan.py [hunk 1]
```diff
@@ -227,7 +227,8 @@ def redraw(self, *_args):
         self.draw_board_contents()
 
     def draw_stone(
-        self, x, y, player, alpha=1, innercol=None, evalcol=None, evalscale=1.0, scale=1.0, ownership=None, loss=None
+            self, x, y, player, alpha=1, innercol=None, evalcol=None, evalscale=1.0, scale=1.0, ownership=None,
+            loss=None, depth=None
```
Comment (sanderland) [inline katrain/gui/badukpan.py:231]: use black -l 120

File: katrain/__main__.py [hunk 2]
```diff
@@ -124,6 +124,8 @@ def __init__(self, **kwargs):
         self.contribute_popup = None
 
         self.pondering = False
+        self.move_num = False
```
Comment (sanderland) [inline katrain/__main__.py:127]: -> show_move_num

**Review by sanderland — CHANGES_REQUESTED** [review]
looks good, some minor comments and please document in the shortcuts list in readme or nobody will be able to find it!

File: katrain/gui/badukpan.py (hunk 1 above)
Comment (xiaoyifang) [reply katrain/gui/badukpan.py:231]: what is this ,column size?

Comment (xiaoyifang) [reply katrain/gui/badukpan.py:231]: If you mean the line width ,the function has exceed 120 .

**Pull request merged**